### PR TITLE
Update AGP to version 7.3.1

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

**Review by emilypgoogle — CHANGES_REQUESTED** [review]
I may have missed something but I tried to make sure all the package names were properly moved to the right place and it looks good. There are some minor indentation issues, and I noticed the placing of the `namespace` attribute is inconsistently before `compiledSdkVersion`. I don't know if that much matters, our build.gradles seem to be in a perpetual state of inconsistent disarray, but I thought I'd mention it.

File: protolite-well-known-types/protolite-well-known-types.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ protobuf {
     generateProtoTasks {
         all().each { task ->
 
-	    task.addIncludeDir(files('fix-javalite/proto'))
+        task.addIncludeDir(files('fix-javalite/proto'))
```
[the remark this answers is not in the document]
Comment (rlazo) [reply protolite-well-known-types/protolite-well-known-types.gradle]: Fixed

**Comment by rlazo** [comment]
> I may have missed something but I tried to make sure all the package names were properly moved to the right place and it looks good. There are some minor indentation issues, and I noticed the placing of the `namespace` attribute is inconsistently before `compiledSdkVersion`. I don't know if that much matters, our build.gradles seem to be in a perpetual state of inconsistent disarray, but I thought I'd mention it.

I've moved the namespace declaration to be on top of compileSdk consistently. Definitely, our build.gradle files are in need of some clean up to be more consistent, but I think it's OK for now.

File: protolite-well-known-types/protolite-well-known-types.gradle [hunk 2]
```diff
@@ -28,7 +28,7 @@ protobuf {
     generateProtoTasks {
         all().each { task ->
 
-	    task.addIncludeDir(files('fix-javalite/proto'))
+	        task.addIncludeDir(files('fix-javalite/proto'))
```
Comment (emilypgoogle) [inline protolite-well-known-types/protolite-well-known-types.gradle:31]: Looks like this got overindented now?

**Review by emilypgoogle — APPROVED** [review]
Single style concern, otherwise everything seems consistently organized and well laid out :+1: all good by me

**Pull request merged**